### PR TITLE
TASK: Use neos-* as composer package type consistently

### PR DESCRIPTION
**Pull request by kdambekalns**
Related to neos/neos-development-collection#1241

### CODE REVIEW

File: TYPO3.Flow/composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "typo3/flow",
-    "type": "typo3-flow-framework",
+    "type": "neos-framework",
     "description": "Flow Application Framework",
     "homepage": "http://flow.typo3.org",
```
Comment (daniellienert) [inline TYPO3.Flow/composer.json]: Should be: https://flow.neos.io/

Comment (kdambekalns) [inline TYPO3.Flow/composer.json]: Thanks. Hint: Edits from maintainers is enabled… :)

**Comment by kdambekalns** [comment]
Re-opened and now Travis picked it up…

File: TYPO3.Flow/composer.json (hunk 1 above)
Comment (daniellienert) [inline TYPO3.Flow/composer.json]: Ahh sorry, forgot about that possibility ...

**Pull request merged**